### DISCUSSION TITLE
74CBTD3861 substitute

### DISCUSSION THREAD

**Discussion by Shadowtrance** [frntc/RAD#18] [General]
Just wondering if there's any drop in substitutes for the 74CBTD3861 as they're a bit of a pain in the butt to get a hold of it seems outside of digikey/mouser which is either expensive postage or spend more and free postage (to australia at least).

Also, the mystery pads and hole next to it, what do they do?
And i guess the mystery pad below that next to the lvc257, what does it do?
I must know the mysteries!! 😂

**Comment by frntc** [comment]
I'm not aware of a drop-in replacement for the 74CBTD3861 :-/ Maybe you're lucky with aliexpress (a quick search revealed vendors). If you happen to find other FET-bus-switches with the same pinout, you can give it a try (let me know). 

That mystery pads were for testing during development, no real use at the moment :-)

**Reply by SensoryTrash** [reply]
hey did you manage to find an alternative or source as digikey are still coming up empty :(

**Comment by pigravity** [comment]
Personally i bought some of 74CBTD3861 on ebay, quite cheap, shipped with no problem.

**Reply by SensoryTrash** [reply]
Thanks do you know by any chance the vendor>?
so many fakes

**Reply by pigravity** [reply]
it was [richde-85](https://www.ebay.com/str/richde85)

**Reply by pigravity** [reply]
but i would still suggest to buy just few pcs for a test :) you never know :)

**Reply by SensoryTrash** [reply]
thank you...

**Reply by SensoryTrash** [reply]
i see he has some more for sale... i will order some e-bay has a good refund policy.... lets hope i don't need to use it.... they look identical to the ones available on Ali express but shipping faster...   and cheaper lol typical
